src/Screens/Gameplay/Components/Map/FocusedMap/Tile.jsx

```diff
@@ -1,5 +1,28 @@
 import './Tile.css';
+import focusedAssets from './focusedMapAssetLib';
 
-export default function Tile() {
-	return <div className='fm-tile'></div>;
+export default function Tile({ tileData }) {
```

Comment:
please make `tileData` an object with keys of `type` and `value`, and render the div's one per type.